infra/scripts/attest_code_reviewer_dispatch.py

```
@@ -332,6 +332,75 @@ def _check_ssot_flow(
+            "`'blocking' if blocking else 'advisory'`"
+        )
+    if any(isinstance(v, ast.Constant) for v in mode_vals):
+        fails.append(
+            f"FAIL: {path} - `mode` is reassigned to a constant, overriding the "
+            "RepoConfig-derived value"
+        )
+    return fails
+
+
+def _check_publish_handlers(
+    f: ast.FunctionDef, path: Path,
```

Ruff (0.15.21)

[warning] Unused function argument: `f`

(ARG001)